File: TCGCollectionManager/src/Frames/Collection.java
package Frames;

import java.util.ArrayList;

public class Collection {
	
	private String name;
	private Game game;
	public ArrayList<Card> cards;
	
	public Collection(String name, Game game) {
		this.name = name;
		this.game = game;
		this.cards = new ArrayList<Card>();
	}
	
	public String getName() {
		return this.name;
	}
	
	public Game getGame() {
		return this.game;
	}
	
	public void addCard(Card card) {
		this.cards.add(card);
	}
}

File: TCG_Manager/collection.py
from datetime import datetime

from game import Game
from card import Card

class Collection:

    def __init__(self, name, game):
        self.name = name
        self.game = game
        self.cards = []
        self.creation_time = datetime.now()

    def add_card(card):
        self.cards.append(card)

    def __eq__(self, x):
        assert type(x) == Collection
        if self.name == x.name:
            return True
        else:
            return False
